docs/source/cn/array_ops.py

```diff
@@ -732,3 +732,197 @@
 
     """,
 )
+
+reset_docstr(
+    oneflow.load,
+    r"""加载一个被 oneflow.save() 保存的对象。
+
+    参数：
+        - **path** (str): 对象所在的路径
+        - **consistent_src_rank** (int, optional): 加载相容 (consistent) 张量的需要的秩 (rank)。被指定时，只有秩与 consistent_src_rank相等的进程才会真正读取 `path` 中的文件，并且被加载对象中的张量会与  placement = `flow.placement('cuda', [consistent_src_rank])` 一致。
```

Comment:
```suggestion
        - **consistent_src_rank** (int, optional): 加载相容 (consistent) 张量的需要的秩 (rank)。被指定时，只有秩与 consistent_src_rank相等的进程才会真正读取 `path` 中的文件，并且被加载对象中的张量会与  placement = `flow.placement('cuda', [consistent_src_rank])` 一致。
```

consistent 不用翻译成“相容”，翻译成“全局”吧，未来也会用 global tensor 的概念代替 consistent tensor 的概念。就算按现在的 consistent 翻译，也翻译成“一致性”比较好。

不过最终我还是腿甲一步到位，翻译成“全局”。其它地方如果又类似的，请一并修改。

Reply:
一并修改完毕